CHANGELOG.md

```diff
@@ -7,7 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [1.0.1] 2020-05-22
+## [1.0.3] 2020-05-22
+
+### Fixed
+
+- Fixed prometheus exporter.
```

Comment:
We should probably be more explicit :joy:

Reply:
It was broken, not it's not anymore (maybe).

That's it :D